docs/serving/samples/hello-world/helloworld-python/README.md

```diff
@@ -177,7 +177,7 @@ cd knative-docs/docs/serving/samples/hello-world/helloworld-python
    > Note: If your URL includes `example.com` then consult the setup instructions for
    > configuring DNS (e.g. with `sslip.io`), or [using a Custom Domain](../serving/using-a-custom-domain).
 
-   === "kubectl"
+=== "kubectl"
 
        ```bash
        kubectl get ksvc helloworld-python  --output=custom-columns=NAME:.metadata.name,URL:.status.url
```

Comment:
unfortunately outdenting this line means the rest of the indentation is wrong and breaks the numbering (the "Now you can make a request.." bullet below should be numbered 2). I think what you need to do is make sure the tab is indented by four spaces (the problem is it was only indented by 3).

```suggestion
    === "kubectl"

        ```bash
        kubectl get ksvc helloworld-python  --output=custom-columns=NAME:.metadata.name,URL:.status.url
        ```

```
(and add an extra space to the start of the line of the rest of the tab group to match. There should be four spaces before the `===` lines, and 8 spaces before all the others).

If it works correctly, it should look like this:

![image](https://user-images.githubusercontent.com/354013/124919600-55d27b00-dfee-11eb-9360-067252499183.png)

(Notice the tabs are indented nicely underneath the numbered bullets, and the bullet beginning "Now you can make a request" starts with the right number)